util/fma_util.py

```diff
@@ -0,0 +1,355 @@
+"""Utilities for loading and exploring FMA dataset, obtained
+from https://github.com/mdeff/fma/"""
```

Comment:
Can you mention the copyright notice as well?

MIT License
Copyright (c) 2016 Michaël Defferrard